pallets/automation-time/src/mock.rs

```diff
@@ -323,3 +324,121 @@ pub fn new_test_ext(state_block_time: u64) -> sp_io::TestExternalities {
 	ext.execute_with(|| Timestamp::set_timestamp(state_block_time));
 	ext
 }
+
+pub fn schedule_task(
```

Comment:
Move these methods from the `tests` module to `mocks` to make it easier to reuse across different test submodules.